specs/VEP-Milestone25-vector-database-ingestion/sequence_diagram.txt

```diff
@@ -0,0 +1,11 @@
+Created using: https://sequencediagram.org/
+
+```
+title RAG data population
+
+VDK->Confluence: Read latest changes from confluence
+VDK<--Confluence: Return confluence pages changed recently along with metadata
+VDK->Embedding Model: Send chunks of text for which embeddings are required
```

Comment:
What is a chunk? How is it chunked? Is that something configurable? Extensible? Predeterminted by the plugin ?

Reply:
Added to the glossary. let me know if you think it is clear or more info should be added.